utilities/infra.py

```diff
@@ -1536,3 +1537,17 @@ def verify_image_info(image_url, generated_pulled_secret, nodes_cpu_architecture
             f"--registry-config={generated_pulled_secret} --filter-by-os={nodes_cpu_architecture}"
         ),
     )
+
+
+def assert_os_version_mismatch_in_vm(vm: utilities.virt.VirtualMachineForTests, expected_os: str) -> None:
```

Comment:
we have something similar here:
https://github.com/RoniKishner/openshift-virtualization-tests/blob/new-vm-with-data-source/tests/infrastructure/golden_images/update_boot_source/test_ssp_common_templates_boot_sources.py#L30

Reply:
can this be part of https://github.com/RoniKishner/openshift-virtualization-tests/blob/new-vm-with-data-source/tests/infrastructure/golden_images/update_boot_source/utils.py

Reply:
since this is used other within infra/golden_images should we cosider moving to local utils file to this folder?